sunpy/net/hek/hek.py

```diff
@@ -18,11 +23,15 @@
 from sunpy.time import parse_time
 from sunpy.util import dict_keys_same, unique
 from sunpy.util.xml import xml_to_dict
+from sunpy import __file__
```

Comment:
I am not sure this is needed here. You can use ``__file__`` without this.

Reply:
Actually the `sunpy.__file__` points at the `sunpy/__init__.py` file while the normal `__file__` points at the current working file.  I removed the import and made the required changes here